packaging/package.sh

```diff
@@ -41,18 +41,16 @@ rm packaging/${PRODUCT_NAME}.dmg || true
 mkdir -p packaging/dmg
 
 # Create directories for the dmg symlinks
-sudo mkdir -m 755 -p /Library/Audio/Plug-Ins/Components && sudo mkdir -m 755 -p /Library/Audio/Plug-Ins/VST3
-ln -s /Library/Audio/Plug-Ins/VST3 "packaging/dmg/Your Mac's VST3 folder"
-cp -r "${ARTIFACTS_PATH}/VST3/${PROJECT_NAME}.vst3" packaging/dmg
+cp -r "${ARTIFACTS_PATH}/${PROJECT_NAME}.app" packaging/dmg
```

Comment:
Do we still want to include a symbolic link to wherever the .app folder should go?

Reply:
yes! that's how most mac dmg app work

Reply:
It looked like it was removed here but I guess it's defined in dmg.json.